### PR TITLE
KIN-323 Add edit about us functionality

### PR DESCRIPTION
**Pull request by Cristianpan**
[KIN-323](https://loopcrack.atlassian.net/browse/KIN-323) Medium [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

The administrator user will be able to make changes to the content of the usage section on the home page using the edit us view in order to keep the section up to date.

### Solution
Implement the logic to update the about us section.

### Requirements checklist

- [x] If the about us text is changed, the record of the home page in the admin panel must be updated with the new text.
- [x] Must validate that the text isn't empty and print an error message in the view if the text is empty.
- [x] For the moment, the files inputs will be ignored and when storing on the database, it will be a hardcoded value 
- [x] It must return a response indicating success on updating the home section or failure in the process 

NOTES: The files functionality will be integrated later

### Testing

Before testing, please run in the console `php spark migrate:refresh` and as next step `php spark db:seed MainTestSeeder`, because I've made a correction about the action of forean keys, I've changed the action from "CASCADE" to "NO ACTION".

- [x] Try to submit the form wit all empty inputs and you can see the error message in the descripion.
- [x] Enter the description data and submit the form and verify that the entered data is saved in the db.
- [x] After saving/updating the data, you can see the "about us" information in the view.

## Demo

[KIN-323]: https://loopcrack.atlassian.net/browse/KIN-323?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/Exceptions/InvalidInputException.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    private array $errors;
+
+    public function __construct(array $errors = [], string $message = '')
+    {
+        $this->errors = $errors;
+        parent::__construct($message);
+    }
+
+    public function getErros()
```
Comment (Mauriciocr207) [inline app/Exceptions/InvalidInputException.php:17]: Change this method to `getErrors`

File: app/Controllers/CtrlAboutUs.php [hunk 2]
```diff
@@ -2,21 +2,53 @@
...
+
+            $response = [
+                'title'   => 'Actualización exitosa',
+                'message' => 'Los datos de la sección sobre nosotros han sido actualizados correctamente',
+                'type'    => 'success',
+            ];
+
+            return redirect()->to('/admin/nosotros')->with('response', $response);
+        } catch (InvalidInputException $th) {
+            return redirect()->to('/admin/nosotros')->withInput()->with('errors', $th->getErros());
```
Comment (Mauriciocr207) [inline app/Controllers/CtrlAboutUs.php:43]: same here

**Review by Mauriciocr207 — APPROVED** [review]
All good, just correct the word I commented

**Review by FernandoJoachin — APPROVED** [review]

**Review by CheloChidoone — APPROVED** [review]
Good job so far @Cristianpan

**Review by CarlosMay7 — APPROVED** [review]

**Review by Oswaldo-Chan — APPROVED** [review]
Great Job @Cristianpan

**Pull request merged**